### PR TITLE
update change log formatting

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

Update the change log formatting to be semantically correct and to stop failing the markdown linter.

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Whew! it sure was a lot of work updating those 35 files by hand. I'm wiped out....

Just kidding, I wrote a script 😄 

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
> stop failing the markdown linter.

we have a markdown linter??

**Comment by mjhenkes** [comment]
1. there is a markdown linter in vs code, i recommend it.
2. the generator has already been updated https://github.com/cerner/generator-terra-module/blob/main/generators/app/templates/CHANGELOG.md

**Comment by emilyrohrbough** [comment]
Oh nice! I'm downloading the linter now!

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
     "wdio-fusion": "THEME=orion-fusion-theme wdio",
     "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
-    "version": "lerna version --no-git-tag-version"
+    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
```
Comment (emilyrohrbough) [inline package.json:103]: Now we REALLY could use a scripts package!

**Pull request merged**